src/operator/nn/mkldnn/mkldnn_lrn-inl.h

```diff
@@ -200,8 +206,15 @@ void MKLDNNLRNBackward(const OpContext &ctx, const LRNParam &param,
   if (req == kNullOp) {
     return;
   }
+
+  // TODO(alex): figure out why in_grad output incorrect when in_data is nchw8c
```

Comment:
Do you have a JIRA ticket to track this TODO? If so, could you please paste it here?